### PR TITLE
move clj and cljs dependencies into dev profile

### PR DESCRIPTION
**Pull request by Quantisan**
Moving dependencies into `:dev` profile so that they don't get bundled into the library.

### CODE REVIEW

**Comment by Quantisan** [comment]
feedback?

File: project.clj [hunk 1]
```diff
@@ -2,11 +2,16 @@
...
             :url "http://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies [[org.clojure/clojure "1.7.0"]
-                 [org.clojure/clojurescript "1.7.228"]
-                 [com.cemerick/clojurescript.test "0.3.0"]]
+
   :plugins [[lein-test-out "0.3.1" :exclusions [org.clojure/clojure]]
             [lein-cljsbuild "1.1.3"]]
+
+  :profiles {:dev {:dependencies
+                   [[org.clojure/test.check "0.9.0"]
```
Comment (nwjsmith) [inline project.clj]: bond doesn't use `test.check`, is this a necessary development dependency?

File: project.clj [hunk 2]
```diff
@@ -2,11 +2,16 @@
...
+                    [org.clojure/clojure "1.7.0"]
+                    [org.clojure/clojurescript "1.7.228"]
```
Comment (nwjsmith) [inline project.clj]: bond requires both Clojure and ClojureScript.

**Comment by neeasade** [comment]
Closing as we no longer support clojurescript -- see https://github.com/circleci/bond/pull/53

**Pull request closed without merging**